apps/src/applab/designElements/button.jsx

```diff
@@ -208,41 +209,70 @@ export default {
   themeValues: {
     backgroundColor: {
       type: 'color',
+      default: color.applab_default_button_background_color,
       classic: color.applab_button_teal,
-      dark: color.yellow
+      orange: color.applab_orange_button_background_color,
+      citrus: color.applab_citrus_button_background_color
     },
     borderRadius: {
+      default: 4,
       classic: 0,
-      dark: 10
+      orange: 0,
+      citrus: 2
     },
     borderWidth: {
+      default: 1,
       classic: 0,
-      dark: 0
+      orange: 2,
+      citrus: 2
     },
     borderColor: {
       type: 'color',
+      default: color.applab_default_button_border_color,
       classic: color.black,
-      dark: color.white
+      orange: color.applab_orange_button_border_color,
+      citrus: color.applab_citrus_button_border_color
     },
     textColor: {
       type: 'color',
+      default: color.applab_default_button_text_color,
       classic: color.white,
-      dark: color.black
+      orange: color.applab_orange_text_color,
+      citrus: color.applab_citrus_text_color
+    },
+    fontFamily: {
+      default: 'Arial Black',
+      classic: 'Arial',
+      orange: 'Verdana',
+      citrus: 'Georgia'
```

Comment:
Is there a way to track the theme styles in one place, instead of sprinkling the styles throughout the different components?

Related: intentional that orange uses 'Verdana' for buttons, but 'Arial' for labels?

Reply:
Agreed, I'm a little worried about this logic being spread out across all of the App Lab design elements.  It seems like it increases the cost of adding a new design element or adding a new theme.  It's possible that having these four themes we are just starting to see the effect of that.  Can we have a more detailed discussion about what a different approach would look like?

Reply:
It is not hard to imagine a consolidation of theme values in a different file. It may be easier for maintenance in the long term. However, there is also a fair amount of code for each component that is interrelated to these themes (when to backfill certain properties with "classic" during deserialize, which properties should be set on creation, etc.)

It may be easier to consolidate after we remove the experiment flag and the older code that runs when the experiment is not enabled

Reply:
@cpirich

> Related: intentional that orange uses 'Verdana' for buttons, but 'Arial' for labels?